### DISCUSSION TITLE
GLIBC update from 2.31 to 2.33…how??

### DISCUSSION THREAD

**Discussion by CGlogger** [catchthatrabbit/coreminer#39] [Miner]
Hi!
Error message:
![D7E46A6E-ECF1-4141-8396-C9B876486F4D](https://user-images.githubusercontent.com/102905641/169648594-0c64d43e-0cca-4719-8b2c-fdc2141c66e6.jpeg)

How can I start the update for the higher version?
I inserted the downloaded file GLIBC 2.33 and moved it to the lib folder with WINSPC….
What is the next step?
Thank you!
![374063AA-3D35-4BCF-92C8-6419FFCF7BB3](https://user-images.githubusercontent.com/102905641/169648551-84fcc430-c28a-4970-88df-c24019cf5657.jpeg)

**Comment by Patrick010** [comment]
Install Debian testing, it has GLIBC 2.33
